libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -358,6 +358,46 @@ export const expoConfig: IConfigCredentials[] = [
   ...pushConfigBase,
 ];
 
+export const pushwooshConfig: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.ApplicationCode,
+    displayName: 'Application Code',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.NotificationTitle,
+    displayName: 'Notification Title',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.Subscribe,
+    displayName: 'Auto Subscribe',
+    type: 'boolean',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.UserId,
+    displayName: 'User ID',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.Tags,
+    displayName: 'Tags',
+    type: 'object',
+    required: true,
```

Comment:
This is not required based on docs.
https://github.com/Pushwoosh/web-push-notifications#usage